prow/cmd/plank/main.go

```diff
@@ -136,9 +137,9 @@ func main() {
 			if err := c.Sync(); err != nil {
 				logger.WithError(err).Error("Error syncing.")
 			}
-			logger.Infof("Sync time: %v", time.Since(start))
+			logger.WithField("duration", fmt.Sprintf("%v", time.Since(start))).Info("Synced")
```

Comment:
Since the `WithField()` takes `interface{}` I'm a bit surprised it's not doing `%v` representation by default -- what does it look like without the `Sprintf()` call?

Reply:
The real value of a time.Duration which is a big float64.

Reply:
ew